dali/python/nvidia/dali/plugin/jax/iterator.py

```diff
@@ -81,7 +81,8 @@ class DALIGenericIterator(_DaliBaseIterator):
     prepare_first_batch : bool, optional, default = True
                 Whether DALI should buffer the first batch right after the creation of the iterator,
                 so one batch is already prepared when the iterator is prompted for the data
-    sharding : ``jax.sharding.Sharding`` compatible object that, if present, will be used to
+    sharding : `jax.sharding.Sharding`
+                `jax.sharding.Sharding` compatible object that, if present, will be used to
```

Comment:
This looked like this before:
![image](https://github.com/NVIDIA/DALI/assets/34919255/972954a1-171a-45d5-8493-291cb03dbbba)